api/models/User.js

```diff
@@ -234,6 +237,46 @@ userSchema.methods.removeSkill = function (skill) {
 	});
 };
 
+/*
+ * Update name for the User object
+ * @param {string} name to update the User object
+ * @returns {User} changed user object
+ */
+
+userSchema.methods.updateName = function (name) {
+	this.name = name;
+
+	return new Promise((resolve, reject) => {
```

Comment:
Both methods repeat this code that saves the currently updated object. Maybe we can move this code to a separate method?

Reply:
In fact, half of all methods in the file has this chunk of code.

Reply:
That’s true, we’ll fix this this a separate PR. So this can be left here for now.

Reply:
(#74)